packages/notification-center/src/components/notification-center/components/user-preference/ChannelPreference.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Text } from './UserPreference';
+import { useNovuThemeProvider } from '../../../../hooks/use-novu-theme-provider.hook';
+import { Switch } from '@mantine/core';
+import { getChannel } from './channels';
+
+export function ChannelPreference({ type, active }) {
+  const channel = getChannel(type);
+  const { theme } = useNovuThemeProvider();
+  const baseTheme = theme?.notificationItem?.unseen;
+
+  return (
+    <div
+      style={{
+        display: 'flex',
+        flexDirection: 'row',
+        alignItems: 'center',
+        justifyContent: 'space-between',
+        padding: '0px',
+        gap: '20px',
+      }}
+    >
+      <div
+        style={{
+          display: 'flex',
+          flexDirection: 'row',
+          alignItems: 'center',
+          justifyContent: 'flex-start',
+          padding: '0px',
+          gap: '15px',
+        }}
+      >
+        {channel.icon}
+        <div
+          style={{
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'flex-start',
+            padding: '0px',
+            gap: '5px',
+          }}
```

Comment:
what do you think about changing to styled-components here so there will be less context here and the components will be more informative?

Reply:
Yes! Forgot to get back to that. Thanks!